tensorflow/compiler/jit/BUILD

```diff
@@ -104,6 +104,23 @@ cc_library(
     alwayslink = 1,
 )
 
+cc_library(
+    name = "xla_amdgpu_device",
+    srcs = ["xla_amdgpu_device.cc"],
+    visibility = [":friends"],
+    deps = [
+        ":jit_compilation_passes",
+        ":xla_device",
+        "//tensorflow/compiler/jit/kernels:xla_launch_op",
+        "//tensorflow/compiler/tf2xla:xla_compiler",
+        "//tensorflow/compiler/tf2xla/kernels:xla_ops",
+        "//tensorflow/compiler/xla/service:gpu_plugin",  # buildcleaner: keep
```

Comment:
Something seems odd here. Recall that at the XLA_GPU level, we agreed it should be possible to link both AMDGPU and NVIDIA GPU support into the same binary. They are mostly orthogonal features.

The dependencies of this device currently look exactly the same as the NVIDIA GPU plugin — you will need to separate them somehow. (e.g., I would expect that you would create an xla/service:amdgpu_plugin)

(It isn't sufficient to use a configure time test to choose which GPU we are targeting, as we discussed.)